.eslintrc.json

```
@@ -13,5 +15,12 @@
 				"allowTernary": true
 			}
 		]
-	}
+	},
+	"parserOptions": {
+        "sourceType": "module"
+	},
+    "env": {
+        "browser": true,
+        "node": true
+    }
 }
```

Maybe not related to the PR, but this file is missing an endline.
```suggestion
}

```